axelor-production/src/main/resources/i18n/messages_fr.csv

```diff
@@ -243,6 +245,7 @@
 "Friday hours","Heures Vendredi",,
 "From","De",,
 "From Date","Date de début",,
+"Fusionned","Fusionné",,
```

Comment:
Fusionned ? Couldn't it me "merged" instead ? Fusionned hardly even exists. If really no better word can be found, write it with a single n : fusioned.